Issue with brand new Scala 2.13

I've update my benchmark project to compare the new `LazyList` vs the old `Stream`, here: <https://github.com/ilaborie/refactorLoops/tree/master/samples-scala>

But I cannot compile the scala part :

```maven
[INFO] --- scala-maven-plugin:4.0.2:compile (default) @ samples-scala ---
[WARNING] Scala library detected 2.13.0 doesn't match scala.compat.version : 2.12.-1
[WARNING]  Expected all dependencies to require Scala version: 2.12.-1
[WARNING]  io.github.ilaborie:samples-scala:1.0-SNAPSHOT requires scala version: 2.13.0
[WARNING] Multiple versions of scala libraries detected!
[INFO] Using incremental compilation using Mixed compile order
[INFO] Compiling 2 Scala sources to /Users/igorlaborie/Documents/Workspaces/playground/refactorLoops/samples-scala/target/classes ...
[ERROR] ## Exception when compiling 2 sources to /Users/igorlaborie/Documents/Workspaces/playground/refactorLoops/samples-scala/target/classes
scala/collection/GenTraversableOnce
xsbt.CompilerInterface.newCompiler(CompilerInterface.scala:22)
java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
java.base/java.lang.reflect.Method.invoke(Method.java:566)
sbt.internal.inc.AnalyzingCompiler.call(AnalyzingCompiler.scala:237)
sbt.internal.inc.AnalyzingCompiler.newCachedCompiler(AnalyzingCompiler.scala:134)
sbt.internal.inc.AnalyzingCompiler.newCachedCompiler(AnalyzingCompiler.scala:121)
sbt.internal.inc.FreshCompilerCache.apply(CompilerCache.scala:78)
sbt.internal.inc.AnalyzingCompiler.compile(AnalyzingCompiler.scala:88)
sbt.internal.inc.MixedAnalyzingCompiler.$anonfun$compile$3(MixedAnalyzingCompiler.scala:82)
scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
sbt.internal.inc.MixedAnalyzingCompiler.timed(MixedAnalyzingCompiler.scala:133)
sbt.internal.inc.MixedAnalyzingCompiler.compileScala$1(MixedAnalyzingCompiler.scala:73)
sbt.internal.inc.MixedAnalyzingCompiler.compile(MixedAnalyzingCompiler.scala:116)
sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1(IncrementalCompilerImpl.scala:307)
sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileInternal$1$adapted(IncrementalCompilerImpl.scala:307)
sbt.internal.inc.Incremental$.doCompile(Incremental.scala:106)
sbt.internal.inc.Incremental$.$anonfun$compile$4(Incremental.scala:87)
sbt.internal.inc.IncrementalCommon.recompileClasses(IncrementalCommon.scala:116)
sbt.internal.inc.IncrementalCommon.cycle(IncrementalCommon.scala:63)
sbt.internal.inc.Incremental$.$anonfun$compile$3(Incremental.scala:89)
sbt.internal.inc.Incremental$.manageClassfiles(Incremental.scala:134)
sbt.internal.inc.Incremental$.compile(Incremental.scala:80)
sbt.internal.inc.IncrementalCompile$.apply(Compile.scala:67)
sbt.internal.inc.IncrementalCompilerImpl.compileInternal(IncrementalCompilerImpl.scala:311)
sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileIncrementally$1(IncrementalCompilerImpl.scala:269)
sbt.internal.inc.IncrementalCompilerImpl.handleCompilationError(IncrementalCompilerImpl.scala:159)
sbt.internal.inc.IncrementalCompilerImpl.compileIncrementally(IncrementalCompilerImpl.scala:238)
sbt.internal.inc.IncrementalCompilerImpl.compile(IncrementalCompilerImpl.scala:69)
sbt_inc.SbtIncrementalCompiler.compile(SbtIncrementalCompiler.java:127)
scala_maven.ScalaCompilerSupport.incrementalCompile(ScalaCompilerSupport.java:284)
scala_maven.ScalaCompilerSupport.compile(ScalaCompilerSupport.java:101)
scala_maven.ScalaCompilerSupport.doExecute(ScalaCompilerSupport.java:83)
scala_maven.ScalaMojoSupport.execute(ScalaMojoSupport.java:555)
org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:156)
org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:148)
org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
org.apache.maven.cli.MavenCli.execute(MavenCli.java:956)
org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
org.apache.maven.cli.MavenCli.main(MavenCli.java:192)
java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
java.base/java.lang.reflect.Method.invoke(Method.java:566)
org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
```

--- a/pom.xml
+++ b/pom.xml
@@ -243,7 +243,7 @@
     <dependency>
       <groupId>org.scala-sbt</groupId>
       <artifactId>zinc_2.12</artifactId>
-      <version>1.2.5</version>
+      <version>1.3.0-M7</version>
     </dependency>
     <dependency>
       <groupId>org.scala-lang.modules</groupId>

--- a/src/main/java/sbt_inc/SbtIncrementalCompiler.java
+++ b/src/main/java/sbt_inc/SbtIncrementalCompiler.java
@@ -96,6 +96,18 @@ public class SbtIncrementalCompiler {
             new T2[] {});
     }
 
+    private PreviousResult previousResult() {
+        Optional<AnalysisContents> analysisContents = analysisStore.get();
+        if (analysisContents.isPresent()) {
+            AnalysisContents analysisContents0 = analysisContents.get();
+            CompileAnalysis previousAnalysis = analysisContents0.getAnalysis();
+            MiniSetup previousSetup = analysisContents0.getMiniSetup();
+            return PreviousResult.of(Optional.of(previousAnalysis), Optional.of(previousSetup));
+        } else {
+            return compiler.emptyPreviousResult();
+        }
+    }
+
     public void compile(List<String> classpathElements, List<File> sources, File classesDirectory,
         List<String> scalacOptions, List<String> javacOptions) {
         List<File> fullClasspath = new ArrayList<>();
@@ -104,7 +116,8 @@ public class SbtIncrementalCompiler {
             fullClasspath.add(new File(classpathElement));
         }
 
-        Inputs inputs = compiler.inputs(fullClasspath.toArray(new File[] {}), // classpath
+        Inputs inputs = compiler.inputs(//
+            fullClasspath.toArray(new File[] {}), // classpath
             sources.toArray(new File[] {}), // sources
             classesDirectory, // classesDirectory
             scalacOptions.toArray(new String[] {}), // scalacOptions
@@ -112,17 +125,11 @@ public class SbtIncrementalCompiler {
             100, // maxErrors
             new Function[] {}, // sourcePositionMappers
             compileOrder, // order
-            compilers, setup, compiler.emptyPreviousResult());
-
-        Optional<AnalysisContents> analysisContents = analysisStore.get();
-        if (analysisContents.isPresent()) {
-            AnalysisContents analysisContents0 = analysisContents.get();
-            CompileAnalysis previousAnalysis = analysisContents0.getAnalysis();
-            MiniSetup previousSetup = analysisContents0.getMiniSetup();
-            PreviousResult previousResult = PreviousResult.of(Optional.of(previousAnalysis),
-                Optional.of(previousSetup));
-            inputs = inputs.withPreviousResult(previousResult);
-        }
+            compilers, // compilers
+            setup, // setup
+            previousResult(), // pr
+            Optional.empty() // temporaryClassesDirectory
+        );
 
         CompileResult newResult = compiler.compile(inputs, logger);
         analysisStore.set(AnalysisContents.create(newResult.analysis(), newResult.setup()));

